### ISSUE TITLE
Registration Service - Command line client - To onboard participant

### ISSUE THREAD

**Issue by cpeeyush** [agera-edc/MinimumViableDataspace#45]
Feature #58 
After #47 

## Description

A command line application needs to be created which can act as registry service client and this application is responsible for onboarding new participant. This command line application internally usages registration service generated Open API specs to interact with backend api. A participant can be onboarded without the need of deploying it first.

In a subsequent version, this client will also need to sign JWS request with participant private key.

 See onboarding spec https://github.com/Metaform/mvd/blob/main/registration-service/registration-service-tech-spec.md#1-onboarding

MVD deployment pipeline should use this command line client in order to onboard dataspace participant.

(Status of onboarding can be access via agera-edc/MinimumViableDataspace#183)

## Acceptance Criteria

- [x] Update registry service ADR agera-edc/MinimumViableDataspace#186 (if needed)
- [x] Registry service command line client available to onboard participant. Client usages Registry service generated OpenAPI specs to interact with api backend.
- [x] <strike>Registry service command line client is able to sign JWS for onboarding.</strike>
- [x] MVD deployment pipeline usages registry service command line client to onboard dataspace participants.
- [x] Testing coverage.
- [x] Update docs and including local development instructions.

## Notes

- A participant can be onboarded without the need of deploying the connector first. This is not currently supported by MVD at the time, since you need to deploy the participant to have the private key generated.
- Could we clarify the expected behavior of the client. Should it busy-loop until onboarding has completed? Or should it have commands for the user to run the client multiple times to query completion?

## Out of scope

Self-Description document for participant and dataspace authority.

**Comment by cpeeyush** [comment]
PR published to have CLI client https://github.com/agera-edc/RegistrationService/pull/2
Upstream PR to have CLI client - https://github.com/eclipse-dataspaceconnector/RegistrationService/pull/7

**Comment by cpeeyush** [comment]
MVD Upstream PR : https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/pull/15
